src/components/subcomponents/posts.js

```diff
@@ -0,0 +1,39 @@
+//ANIMATION TEST
+import React                       from "react";
+import TextTransition, { presets } from "react-text-transition";
+
+const post = [
```

Comment:
Would be a good idea to render the text in app.js and pass it down as this is how it will be handled once the CMS is in place